README.md

```diff
@@ -776,6 +780,169 @@ You'll see 504 gateway timeout 10 sec after you send the request
 ```
 See also [Request Timeouts](https://istio.io/latest/docs/tasks/traffic-management/request-timeouts/)
 
+#### Connect Timeout
+The configuration below set 4 sec tcp connection timeout for a request to the non-existent service.
+
+Prepare a non-existent service and check the connect timeout.
+The ip in the Endpoints resource should be a non-existent IP address.
+
+```yaml
+# kubectl apply -f manifests/non-existent-service.yaml -n testns1
+
+kubectl apply -n testns1 -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: non-existent-srevice
```

Comment:
Please fix typo like below:
`srevice` -> `service`

Reply:
Thanks for comment.
Fixed it.
https://github.com/yokawasa/istio-sandbox/pull/3/commits/700204713aaebb9e29cb3772f2d5e70772cb6b80